youtube-client/src/components/Card.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import Card from '@mui/material/Card';
```

Comment:
due to webpack 2+ you could combine mui imports due to [tree shaking mechanism](https://webpack.js.org/guides/tree-shaking/)